src/app/App.js

```diff
@@ -0,0 +1,51 @@
+import {useState} from 'react';
+import './App.css';
+import Counter from '../shared/ui/Counter/Counter';
+import SearchForm from '../shared/ui/SearchForm/SearchForm';
+import SelectTabs from '../shared/ui/SelectTabs/SelectTabs';
+
+
+const genres = [
+  { id: "all", name: "genre", value: "ALL" },
+  { id: "doc", name: "genre", value: "DOCUMENTARY" },
+  { id: "com", name: "genre", value: "COMEDY" },
+  { id: "horror", name: "genre", value: "HORROR" },
+  { id: "crime", name: "genre", value: "CRIME" },
+];
+
+const App = () => {
+  const [searchValue, setSearchValue] = useState("");
+  const [selectedGenre, setSelectedGenre] = useState("ALL");
+
+  const handleSearchQueryChange = (event) => {
+    const value = event.target.value;
+    setSearchValue(value);
```

Comment:
there is no need to put `event.target.value` into a separate variable, we need variables in most cases for:
1. Give a human-readable name for an expression that can be not obvious (e.g. 
`const isAllowedAccess = userAge >= 18 && userHasPermission && !isBanned;`

```
if(isAllowedAccess){
 ... 
}
```
2. We need to reuse repeatable value in several places to manage it in one place (e.g. 

`const primaryColor = '#424242'; `

```
const headerStyle = {
  backgroundColor: primaryColor,
};
```

```
const buttonStyle = {
  backgroundColor: primaryColor,
};
```

In our example `event.target.value` is clear, so let's put it directly 
setSearchValue(event.target.value);

Reply:
1. Actually firstly I tried setSearchValue(event.target.value); but I don't know why the value was undefined , and it threw an error, but the thing is when I consoled event.target.value before setSearchValue(event.target.value); the value was okay, so I guessed the problem was it didn't deliver to change the value and I encountered this problem for the first time.

However this time it worked.

done

Reply:
2. done

Reply:
sorry for confusing you, I didn't mean to put colors to variables, it was just my example where we need to use variables, Could you please revert your changes connected to styles and use CSS as it was before, if it's needed I'll explain how we can resue colors later using SCSS

Reply:
> sorry for confusing you, I didn't mean to put colors to variables, it was just my example where we need to use variables, Could you please revert your changes connected to styles and use CSS as it was before, if it's needed I'll explain how we can resue colors later using SCSS

Reply:
okay ,  my current work project is with scss and variables,, and I thought maybe we aim to that style

Reply:
done